src/multilevel_bisect/initial_partitioning.cpp

```diff
@@ -27,6 +27,9 @@ long initial_partitioning(pmondriaan::hypergraph& H,
     // pmondriaan::interval labels = {0,1};
     // bisect_random(H, max_weight_0, max_weight_1, 0, H.size(), labels, rng);
 
+    break_triples(H);

```

Comment:
I'm entirely unsure where to use it because I found it to be very dependent on the structure of the hypergraph. The main reason to use it here is that there are many cases where we may be able to use graph partitioning for the initial partitioning phase after this method. Another reason is that the hyperedge sizes tend to be smallest here and so we may find many of size 2 and 3.